app/post/post_test.go

```diff
@@ -0,0 +1,69 @@
+// Copyright Tharsis Labs Ltd.(Evmosii)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+package post_test
+
+import (
+	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	"github.com/evmos/evmos/v15/app/post"
+)
+
+func (s *PostTestSuite) TestBurnDecorator() {
```

Comment:
nit: it's not really testing the decorator but rather the handler options

```suggestion
func (s *PostTestSuite) TestPostHandlerOptions() {
```